src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/submission_details.cljs

```diff
@@ -63,7 +64,18 @@
                                        (map (fn [message]
                                               [:div {} message])
                                          message-list)])}
-                 {:header "Workflow ID" :starting-width 300}]
+                 {:header "Workflow ID" :starting-width 300
+                  :content-renderer
+                  (fn [workflow]
+                   (let [submission-id (:submission-id props) 
+                         bucketName (:bucketName props)
+                         inputs (second (second (first (workflow "inputResolutions"))))
+                         inputNames (string/split inputs ".")
+                         workflowName (first inputNames) 
+                         workflowId (workflow "workflowId")]
```

Comment:
By contrast, this one is a judgement call, since it may make sense sometimes to use camel case to match the JSON variable name.